garden_ai/gardens.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import annotations
```

Comment:
[comment is not actually about this line of code]

A thought for follow-up work: I think we'll want more subdirectories under garden_ai. At least a directory for the pydantic models. It's awkward to have those at the same level as the autogenerated datacite file.

Reply:
yeah, I was noticing that in a big way looking at the generated documentation too, it was pretty ugly -- though on one hand, the `DataciteSchema` object and `Garden` object are both pydantic models, so it might be odd for only one of them to live in a `models/` subdirectory, idk. V{>0}'s problem.